modules/31_fossil/timeDepGrades/input/files

```diff
@@ -1,7 +1,7 @@
 p31_grades_coal.cs4r
 p31_grades_gas.cs4r
 p31_grades_oil.cs4r
-f31_decoffset.cs4r
-f31_Xport.cs4r
-f31_extraseed.cs4r
+*f31_decoffset.cs4r
+*f31_Xport.cs4r
+*f31_extraseed.cs4r
```

Comment:
maybe delete those lines?

Reply:
Well, as long as they are still called here, I think it is better just to comment them out: https://github.com/remindmodel/remind/blob/eb46563ccaccb587e1c44876a0df879428c7f7c9/modules/31_fossil/timeDepGrades/preloop.gms#L56

Reply:
but they do not exist...and if yomeone would provide them, they would not be dirstributed as long as they would be commented in in this file. So, I think, it is better to have no outcommented code. If this will be reactivated, there are anyway code adjsutments necessary.